[
   {
      "body" : "Are you sure the value is wrong? What sort of result are you expecting?",
      "created_at" : "2016-03-15T07:10:13Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/7690#issuecomment-196693647",
      "id" : 196693647,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/7690",
      "updated_at" : "2016-03-15T07:10:13Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/196693647",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/17022881?v=3",
         "events_url" : "https://api.github.com/users/kirkalx/events{/privacy}",
         "followers_url" : "https://api.github.com/users/kirkalx/followers",
         "following_url" : "https://api.github.com/users/kirkalx/following{/other_user}",
         "gists_url" : "https://api.github.com/users/kirkalx/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/kirkalx",
         "id" : 17022881,
         "login" : "kirkalx",
         "organizations_url" : "https://api.github.com/users/kirkalx/orgs",
         "received_events_url" : "https://api.github.com/users/kirkalx/received_events",
         "repos_url" : "https://api.github.com/users/kirkalx/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/kirkalx/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/kirkalx/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/kirkalx"
      }
   },
   {
      "body" : "Did you try `listunspent 0` to see if there is anything suspicious. ",
      "created_at" : "2016-03-15T11:19:43Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/7690#issuecomment-196769973",
      "id" : 196769973,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/7690",
      "updated_at" : "2016-03-15T11:19:43Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/196769973",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/6399679?v=3",
         "events_url" : "https://api.github.com/users/MarcoFalke/events{/privacy}",
         "followers_url" : "https://api.github.com/users/MarcoFalke/followers",
         "following_url" : "https://api.github.com/users/MarcoFalke/following{/other_user}",
         "gists_url" : "https://api.github.com/users/MarcoFalke/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/MarcoFalke",
         "id" : 6399679,
         "login" : "MarcoFalke",
         "organizations_url" : "https://api.github.com/users/MarcoFalke/orgs",
         "received_events_url" : "https://api.github.com/users/MarcoFalke/received_events",
         "repos_url" : "https://api.github.com/users/MarcoFalke/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/MarcoFalke/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/MarcoFalke/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/MarcoFalke"
      }
   },
   {
      "body" : "Yes, 100%. I mean. I've never had this problem before in a year and the volume of btc is exactly the same. It's been stuck at 255+ BTC for about 5 days now all of a sudden. And there is no way 255 BTC is incoming to an adress. Or that they havent been confirmed by now. When it first happened listtransactions didn't list a big tranasction incoming either.\r\n\r\nFor example: When someone sends 1 BTC getunconfirmedbalance will show as 256 BTC and then go down to 255 BTC again once its confirmed.\r\n\r\nYes MarcoFalke, Iv'e tried it. The weird thing about this command is that it lists lots of transactions saying \"confirmations: 0\" which already have thousands of confirmations. Some dated 2015 with over 25.000+ confirmations and still says \"confirmations: 0\". Like this one: \r\nhttps://www.blocktrail.com/BTC/address/16F4Bzkk6xaFktqSopACKn18rCRWCWgdW3/transactions\r\n    \"txid\": \"a34a18358937a5b9a9fff8281491de57a9c7942217d43bb469eef5d5113d50fc\",\r\n    \"vout\": 1,\r\n    \"address\": \"16F4Bzkk6xaFktqSopACKn18rCRWCWgdW3\",\r\n    \"account\": \"usedaddress\",\r\n    \"scriptPubKey\": \"76a91439811d6af01a4337137356cb88488f9f1b6348ed88ac\",\r\n    \"amount\": 0.23509566,\r\n    \"confirmations\": 0,\r\n    \"spendable\": true\r\nThis one shouldn't be there at all, right?\r\n\r\nIt all started around 3 days after I've upgraded from 0.11 to 0.12. My wallet is around 230 MB. Should I do rescan/reindex? I rather not since it takes lots of time ;(\r\n",
      "created_at" : "2016-03-15T12:30:46Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/7690#issuecomment-196798489",
      "id" : 196798489,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/7690",
      "updated_at" : "2016-03-15T12:31:20Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/196798489",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/17840310?v=3",
         "events_url" : "https://api.github.com/users/BitcoinUser2016/events{/privacy}",
         "followers_url" : "https://api.github.com/users/BitcoinUser2016/followers",
         "following_url" : "https://api.github.com/users/BitcoinUser2016/following{/other_user}",
         "gists_url" : "https://api.github.com/users/BitcoinUser2016/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/BitcoinUser2016",
         "id" : 17840310,
         "login" : "BitcoinUser2016",
         "organizations_url" : "https://api.github.com/users/BitcoinUser2016/orgs",
         "received_events_url" : "https://api.github.com/users/BitcoinUser2016/received_events",
         "repos_url" : "https://api.github.com/users/BitcoinUser2016/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/BitcoinUser2016/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/BitcoinUser2016/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/BitcoinUser2016"
      }
   },
   {
      "body" : "Conflicted transactions count toward the unconfirmed balance, IIRC.",
      "created_at" : "2016-03-15T12:37:02Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/7690#issuecomment-196800687",
      "id" : 196800687,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/7690",
      "updated_at" : "2016-03-15T12:37:02Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/196800687",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/6399679?v=3",
         "events_url" : "https://api.github.com/users/MarcoFalke/events{/privacy}",
         "followers_url" : "https://api.github.com/users/MarcoFalke/followers",
         "following_url" : "https://api.github.com/users/MarcoFalke/following{/other_user}",
         "gists_url" : "https://api.github.com/users/MarcoFalke/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/MarcoFalke",
         "id" : 6399679,
         "login" : "MarcoFalke",
         "organizations_url" : "https://api.github.com/users/MarcoFalke/orgs",
         "received_events_url" : "https://api.github.com/users/MarcoFalke/received_events",
         "repos_url" : "https://api.github.com/users/MarcoFalke/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/MarcoFalke/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/MarcoFalke/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/MarcoFalke"
      }
   },
   {
      "body" : "That transaction is a double spend that did not make it into the block\nchain. 0.12 deals differently with those; see the release notes for more\ninformation.\n\nRunning with -rescan once will fix the problem, by explicitly marking it as\nconflicting with the block chain. After that, you can choose to remove such\ntransactions from the wallet with `abandontransaction`.\nOn Mar 15, 2016 13:30, \"BitcoinUser2016\" <notifications@github.com> wrote:\n\n> Yes, 100%. I mean. I've never had this problem before in a year and the\n> volume of btc is exactly the same. It's been stuck at 255+ BTC for about 5\n> days now all of a sudden. And there is no way 255 BTC is incoming to an\n> adress. Or that they havent been confirmed by now. When it first happened\n> listtransactions didn't list a big tranasction incoming either.\n>\n> For example: When someone sends 1 BTC getunconfirmedbalance will show as\n> 256 BTC and then go down to 255 BTC again once its confirmed.\n>\n> Yes MarcoFalke, Iv'e tried it. The weird thing about this command is that\n> it lists lots of \"confirmations: 0\"-transactions which already have\n> thousands of confirmations. Some dated 2015 with over 25.000+ confirmations\n> and still says \"confirmations: 0\". Like this one:\n>\n> https://www.blocktrail.com/BTC/address/16F4Bzkk6xaFktqSopACKn18rCRWCWgdW3/transactions\n> \"txid\": \"a34a18358937a5b9a9fff8281491de57a9c7942217d43bb469eef5d5113d50fc\",\n> \"vout\": 1,\n> \"address\": \"16F4Bzkk6xaFktqSopACKn18rCRWCWgdW3\",\n> \"account\": \"usedaddress\",\n> \"scriptPubKey\": \"76a91439811d6af01a4337137356cb88488f9f1b6348ed88ac\",\n> \"amount\": 0.23509566,\n> \"confirmations\": 0,\n> \"spendable\": true\n> This one shouldn't be there at all, right?\n>\n> It all started around 3 days after I've upgraded from 0.11 to 0.12. My\n> wallet is around 230 MB. Should I do rescan/reindex? I rather not since it\n> takes lots of time ;(\n>\n> Ã¢ÂÂ\n> You are receiving this because you are subscribed to this thread.\n> Reply to this email directly or view it on GitHub:\n> https://github.com/bitcoin/bitcoin/issues/7690#issuecomment-196798489\n>\n",
      "created_at" : "2016-03-15T12:41:14Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/7690#issuecomment-196801801",
      "id" : 196801801,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/7690",
      "updated_at" : "2016-03-15T12:41:14Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/196801801",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/548488?v=3",
         "events_url" : "https://api.github.com/users/sipa/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sipa/followers",
         "following_url" : "https://api.github.com/users/sipa/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sipa/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sipa",
         "id" : 548488,
         "login" : "sipa",
         "organizations_url" : "https://api.github.com/users/sipa/orgs",
         "received_events_url" : "https://api.github.com/users/sipa/received_events",
         "repos_url" : "https://api.github.com/users/sipa/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sipa/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sipa/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sipa"
      }
   },
   {
      "body" : "Forgot to thank you all for your help guys.\r\n\r\nHow and why does a transraction become conflicted? And how could this happen in a minute? Because I remember checking the confirmed balance 30 mins or so prior to this. And suddenly it started saying 255.359391 BTC+. \r\n\r\nWill do a rescan then. Any idea how long this can take?\r\nAlso: Is it dangerous to keep running my bitcoind while having these \"issues\"?",
      "created_at" : "2016-03-15T12:47:23Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/7690#issuecomment-196803468",
      "id" : 196803468,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/7690",
      "updated_at" : "2016-03-15T12:47:23Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/196803468",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/17840310?v=3",
         "events_url" : "https://api.github.com/users/BitcoinUser2016/events{/privacy}",
         "followers_url" : "https://api.github.com/users/BitcoinUser2016/followers",
         "following_url" : "https://api.github.com/users/BitcoinUser2016/following{/other_user}",
         "gists_url" : "https://api.github.com/users/BitcoinUser2016/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/BitcoinUser2016",
         "id" : 17840310,
         "login" : "BitcoinUser2016",
         "organizations_url" : "https://api.github.com/users/BitcoinUser2016/orgs",
         "received_events_url" : "https://api.github.com/users/BitcoinUser2016/received_events",
         "repos_url" : "https://api.github.com/users/BitcoinUser2016/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/BitcoinUser2016/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/BitcoinUser2016/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/BitcoinUser2016"
      }
   },
   {
      "body" : "You always had these conflicting transactions in your wallet. Up to 0.11\nthey were just not counted towards your balance, as a heuristic was\ndetermining they couldn't be accepted (they were not accepted into your own\nmempool). Due to the large changes that the mempool underwent in 0.12, that\nheuristic is no longer usable, so the wallet now tracks conflicts by\nitself. A rescan will let your wallet go through history and gather that\ninformation.\n",
      "created_at" : "2016-03-15T12:56:10Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/7690#issuecomment-196805767",
      "id" : 196805767,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/7690",
      "updated_at" : "2016-03-15T12:56:10Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/196805767",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/548488?v=3",
         "events_url" : "https://api.github.com/users/sipa/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sipa/followers",
         "following_url" : "https://api.github.com/users/sipa/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sipa/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sipa",
         "id" : 548488,
         "login" : "sipa",
         "organizations_url" : "https://api.github.com/users/sipa/orgs",
         "received_events_url" : "https://api.github.com/users/sipa/received_events",
         "repos_url" : "https://api.github.com/users/sipa/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sipa/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sipa/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sipa"
      }
   },
   {
      "body" : "Thanks!!",
      "created_at" : "2016-03-15T13:00:28Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/7690#issuecomment-196806611",
      "id" : 196806611,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/7690",
      "updated_at" : "2016-03-15T13:00:28Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/196806611",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/17840310?v=3",
         "events_url" : "https://api.github.com/users/BitcoinUser2016/events{/privacy}",
         "followers_url" : "https://api.github.com/users/BitcoinUser2016/followers",
         "following_url" : "https://api.github.com/users/BitcoinUser2016/following{/other_user}",
         "gists_url" : "https://api.github.com/users/BitcoinUser2016/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/BitcoinUser2016",
         "id" : 17840310,
         "login" : "BitcoinUser2016",
         "organizations_url" : "https://api.github.com/users/BitcoinUser2016/orgs",
         "received_events_url" : "https://api.github.com/users/BitcoinUser2016/received_events",
         "repos_url" : "https://api.github.com/users/BitcoinUser2016/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/BitcoinUser2016/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/BitcoinUser2016/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/BitcoinUser2016"
      }
   }
]
